pepper-apis/src/main/java/org/broadinstitute/ddp/export/DataExportCoordinator.java

```diff
@@ -0,0 +1,292 @@
+                LOG.info("Extracted {} participants for study {}", participants.size(), studyDto.getGuid());
+                if (indices.contains(ElasticSearchIndexType.PARTICIPANTS_STRUCTURED)) {
+                    try {
+                        exporter.exportToElasticsearch(handle, studyDto, activities, participants, true);
+                    } catch (Exception e) {
+                        LOG.error("Error while running {} elasticsearch export for study {}, continuing",
+                                ElasticSearchIndexType.PARTICIPANTS_STRUCTURED, studyGuid, e);
+                        batchSuccess = false;
+                    }
+                }
+                if (indices.contains(ElasticSearchIndexType.PARTICIPANTS)) {
+                    try {
+                        exporter.exportToElasticsearch(handle, studyDto, activities, participants, false);
+                    } catch (Exception e) {
+                        LOG.error("Error while running {} elasticsearch export for study {}, continuing",
+                                ElasticSearchIndexType.PARTICIPANTS, studyGuid, e);
+                        batchSuccess = false;
+                    }
+                }
+                if (indices.contains(ElasticSearchIndexType.USERS)) {
+                    try {
+                        exporter.exportUsersToElasticsearch(handle, studyDto, batch);
+                    } catch (Exception e) {
+                        LOG.error("Error while running {} elasticsearch export for study {}, continuing",
+                                ElasticSearchIndexType.USERS, studyGuid, e);
+                        batchSuccess = false;
+                    }
+                }
+                return batchSuccess;
+            });
+            success = success && runSuccess;
+
+            fetched += fetchedSize;
+        }
+        LOG.info("Processed {} participants for study {}", fetched, studyDto.getGuid());
+        return success;
+    }
+
+    private boolean runCsvExports(StudyDto studyDto, List<ActivityExtract> activities) {
+        String studyGuid = studyDto.getGuid();
+        try {
+            LOG.info("Running csv export for study {}", studyGuid);
+            long start = Instant.now().toEpochMilli();
+            var iterator = new PaginatedParticipantIterator(studyDto, batchSize);
+            exportStudyToGoogleBucket(studyDto, exporter, csvBucket, activities, iterator);
+            long elapsed = Instant.now().toEpochMilli() - start;
+            LOG.info("Finished csv export for study {} in {}s", studyGuid, elapsed / 1000);
+            return true;
+        } catch (Exception e) {
+            LOG.error("Error while running csv export for study {}, continuing", studyGuid, e);
+            return false;
+        }
+    }
+
+    boolean exportStudyToGoogleBucket(StudyDto studyDto, DataExporter exporter, Bucket bucket,
```

Comment:
Most of the orchestration here for csv exports was moved over from `StudyDataExportJob`.